src/core/default/commands/function/logs.py

```diff
@@ -63,26 +86,77 @@ def command(self, *args, **kwargs) -> None:
             return
 
         cloud_watch_client = client("logs")
-        print(cloud_watch_group_name)
         log_streams_rv = cloud_watch_client.describe_log_streams(
             logGroupName=cloud_watch_group_name,
             orderBy="LastEventTime",
         )
 
         streams = _get_needed_streams(log_streams_rv.get("logStreams"))
-
-        for stream in streams:
-            response = cloud_watch_client.get_log_events(
+        if query_val is None:
+            for stream in streams:
+                response = cloud_watch_client.get_log_events(
+                    logGroupName=cloud_watch_group_name,
+                    logStreamName=stream,
+                    limit=limit_val,
+                    startFromHead=False if tail_val else True,
+                )
+                if tail_val:
+                    next_token = response.get("nextForwardToken")
+                else:
+                    next_token = response.get("nextBackwardToken")
+                prev_token = ""
+                while next_token != prev_token:
+                    for event in response.get("events"):
+                        self.stdout.write(
+                            f"{datetime.datetime.fromtimestamp(event.get('timestamp') / 1000).strftime('%Y-%m-%d %H:%M:%S')} - {event.get('message')}"
+                        )
+                    response = cloud_watch_client.get_log_events(
+                        logGroupName=cloud_watch_group_name,
+                        logStreamName=stream,
+                        limit=limit_val,
+                        startFromHead=False if tail_val else True,
+                        nextToken=next_token
+                    )
+                    prev_token = next_token
+                    if tail_val:
+                        next_token = response.get("nextForwardToken")
+                    else:
+                        next_token = response.get("nextBackwardToken")
+        else:
+            if start_time_val is None:
+                self.stdout.write('--start_time value is needed when using the query option')
+            elif end_time_val is None:
```

Comment:
Lets make end time default to the current system time. I feel like this could help ergonomics slightly because I think most people by default would want to query from a start time to the current time. 

Not 100% sure on this though just a thought that I had.

Reply:
Let me know if you think it should stay as a required param